media-proxy/src/shm_memif_rdma.c

```diff
@@ -57,10 +57,10 @@ int rx_rdma_on_disconnect(memif_conn_handle_t conn, void *priv_data)
     }
 
     // release session
-    if (!rx_ctx->shm_ready) {
+    if (!atomic_load(&rx_ctx->shm_ready)) {
         return 0;
     }
-    rx_ctx->shm_ready = 0;
+    atomic_store(&rx_ctx->shm_ready, false);
```

Comment:
```suggestion
    atomic_store_explicit(&rx_ctx->shm_ready, false, memory_order_release);
```

Reply:
Here I used relaxed instead of release. We don't modify any memory before this store operation, so there is no need to barrier.